propagator/xray/opentelemetry-propagator-xray.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-api', '~> 0.17.0'
 
-  spec.add_development_dependency 'bundler', '>= 1.17'
+  spec.add_development_dependency 'bundler', '~> 1.17'
```

Comment:
```suggestion
  spec.add_development_dependency 'bundler', '>= 1.17'
```
It looks like this change breaks CI.

Reply:
It might be because mentioned bundler version is too old. I bumped it up a bit, let's see if it gets picked up. Otherwise I'll revert the file.

Reply:
I get this complaint when I build the gem locally 
`jruby -S gem build opentelemetry-propagator-xray.gemspec`

![image](https://user-images.githubusercontent.com/8445726/116494949-8a320880-a8d4-11eb-98b1-b11c86b46ccb.png)

Reply:
I believe this warning can safely be ignored. Libraries should generally be open-ended in their version dependencies until the authors know for certain that use with a later version causes problems.

Reply:
Another option would be to move all the gems in this repository to require Bundler `~> 2.0` for development and test, which could be a good change but is out-of-scope for this trace header parsing fix.

Reply:
reverted. Thanks